### PR TITLE
BAU: update paymentCreated event contract test

### PR DESCRIPTION
**Pull request by jankowiakmaria**
validation of relevant transaction properties (filled based on the paymentCreated event)

### CODE REVIEW

File: src/test/java/uk/gov/pay/ledger/pact/PaymentCreatedEventQueueContractTest.java [hunk 1]
```diff
@@ -57,19 +62,24 @@ public MessagePact createPaymentCreatedEventPact(MessagePactBuilder builder) {
...
+        assertThat(transaction.isPresent(), is(true));
+        assertThat(transaction.get().getExternalId(), is(externalId));
+        assertThat(transaction.get().getAmount(), is(1000L));
+        assertThat(transaction.get().getDescription(), is("a description"));
+        assertThat(transaction.get().getReference(), is("aref"));
+        assertThat(transaction.get().getGatewayAccountId(), is(gatewayAccountId));
+        assertThat(transaction.get().getTotalAmount(), is(1069L));
+        assertThat(transaction.get().getTransactionDetails(), containsString("\"return_url\": \"https://example.org\""));
+        assertThat(transaction.get().getTransactionDetails(), containsString("\"payment_provider\": \"sandbox\""));
+        assertThat(transaction.get().getTransactionDetails(), containsString("\"corporate_surcharge\": 69"));
```
Comment (kbottla) [inline src/test/java/uk/gov/pay/ledger/pact/PaymentCreatedEventQueueContractTest.java]: Is `corporate_surcharge`/`total_amount` available for `PaymentCreated` event ? Or is it meant for `PaymentDetailsEntered` ?

Comment (jankowiakmaria) [reply src/test/java/uk/gov/pay/ledger/pact/PaymentCreatedEventQueueContractTest.java]: At the moment they're in both. I guess they should be cleaned up.

Comment (jankowiakmaria) [reply src/test/java/uk/gov/pay/ledger/pact/PaymentCreatedEventQueueContractTest.java]: Removed in https://github.com/alphagov/pay-ledger/compare/91af1595970716392e91434e438e36d98e73614c..30f130cd6ccd4868ee2d419bd46e23e2f3849c3b

Comment (kbottla) [reply src/test/java/uk/gov/pay/ledger/pact/PaymentCreatedEventQueueContractTest.java]: 👍 Yeah it needs to be cleaned up

**Pull request merged**